api/Element.json

```diff
@@ -8375,13 +8466,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/tabStop",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": false
```

Comment:
Looks like https://developer.mozilla.org/en-US/docs/Web/API/Element/tabStop was never shipped. We should remove this entry, but not in this PR.